#!/usr/bin/env python
PACKAGE = "static_planner"

from dynamic_reconfigure.parameter_generator_catkin import ParameterGenerator, int_t, double_t, bool_t

gen = ParameterGenerator()

gen.add("lethal_cost", int_t, 0, "Lethal Cost", 253, 1, 255)
gen.add("neutral_cost", int_t, 0, "Neutral Cost", 50, 1, 255)
gen.add("cost_factor", double_t, 0, "Factor to multiply each cost from costmap by", 4.0, 0.01, 5.0)
gen.add("rough_length", double_t, 0, "Factor to decrease size of costmap though rough the Index", 0.3, 0, 2.0)

exit(gen.generate(PACKAGE, "static_planner", "StaticPlanner"))
